core/platforms/packaging/bareos-Univention_4.0.dsc

```diff
@@ -8,5 +8,4 @@ Homepage: http://www.bareos.org/
 Standards-Version: 3.9.4
 Build-Depends: acl-dev, bc, chrpath, debhelper (>= 7.0.50~), dpkg-dev (>= 1.13.19), git-core, libacl1-dev, libcap-dev, libfastlz-dev, libjansson-dev, liblzo2-dev, qtbase5-dev, libreadline-dev, libssl-dev, libwrap0-dev, libx11-dev, libsqlite3-dev, libmysqlclient-dev, libpq-dev (>= 8.4), logrotate, lsb-release, mtx, ncurses-dev, openssl, pkg-config, po-debconf (>= 0.8.2), python-dev, ucslint, univention-config-dev, wget, zlib1g-dev, postgresql-server-dev-all, cmake
 Build-Conflicts: python2.2-dev, python2.3, python2.4, qt3-dev-tools
-DEBTRANSFORM-RELEASE: 1
```

Comment:
Are you aware, that this setting is required to build the packages via openbuildservice (OSB)? The same applies to the *.changes files.

Reply:
Yes. I'm working on the Jenkins build-infrastructure to configure OBS on a way that doen't require these files anymore. But thanks for noticing :)